cmd/init.go

```diff
@@ -77,39 +77,55 @@ var initCmd = &cobra.Command{
 
 		return c.Run()
 	},
-	Long: `Initialize a ksonnet project in a new directory,` + " `app-name`" + `. This process
-consists of two steps:
-
-1. Generating ksonnet-lib. Users can set flags to generate the library based on
-   a variety of data, including server configuration and an OpenAPI
-   specification of a Kubernetes build. By default, this is generated from the
-   capabilities of the cluster specified in the cluster of the current context
-   specified in` + " `$KUBECONFIG`" + `.
-2. Generating the following tree in the current directory.
-
-        app-name/
-          .gitignore     Default .gitignore; can customize VCS
-          .ksonnet/      Metadata for ksonnet
-          environments/
-            default/      [Default generated environment]
-          components/    Top-level Kubernetes objects defining application
-          lib/           user-written .libsonnet files
-          vendor/        part libraries, prototypes
+	Long: `Initialize a ksonnet application in a new directory,` + " `app-name`" + `.
+
+This command will generate all the project scafolding required to begin creating and
+deploying components to Kubernetes clusters.
+
+ksonnet applications are initialized based on your current cluster configurations,
+as defined in your` + " `$KUBECONFIG` " + `environment variable. See examples
+of how to customize configurations in the *Examples* section below.
+
+Creating a ksonnet application will result in the following directory tree.
+
+    app-name/
+      .ksonnet/      Metadata for ksonnet
+      app.yaml       Application specifications, ex: name, api version
+      components/    Top-level Kubernetes objects defining the application
+      environments/  Kubernetes cluster definitions
+        default/     Default environment, initialized from the current kubeconfig
+          .metadata/ [1]
+      lib/           user-written .libsonnet files
+      vendor/        part libraries, prototypes
+
+To begin populating your ksonnet application, visit` + " `ks generate` " + `.
+
+[1] Each environment uses a specific version of ksonnet-lib. Users can set flags
+to generate the library based on a variety of data, including server
+configuration and an OpenAPI specification of a Kubernetes build. By default,
+this is generated from the capabilities of the cluster specified in the cluster
+of the current context specified in` + " `$KUBECONFIG`" + `.
 `,
-	Example: `# Initialize ksonnet application, using the capabilities of live cluster
-# specified in the $KUBECONFIG environment variable (specifically: the
-# current context) to generate 'ksonnet-lib'.
+	Example: `# Initialize a ksonnet application, using the live cluster specified in
```

Comment:
Feel free to disagree, but I think the following is a bit more clear:
```
Initialize a ksonnet application, based on cluster info from the active kubeconfig file 
(specified by environment variable $KUBECONFIG). More specifically, the current context is
used.
```

Reply:
more clear in the sense that `$KUBECONFIG` doesn't inherently contain cluster info, but rather points to a file